test/support/concurrent_queue.h

```diff
@@ -14,6 +14,7 @@
 #include <mutex>
 #include <deque>
 #include <condition_variable>
+#include <list>
```

Comment:
I think this whole header is no longer needed. And should be removed, along its include in the inline_backend